Code/Engine/Texture/TexConv/TexConvProcessor.h

```diff
@@ -48,14 +48,7 @@ class EZ_TEXTURE_DLL ezTexConvProcessor
   ezResult PremultiplyAlpha(ezImage& image) const;
   ezResult DilateColor2D(ezImage& img) const;
   ezResult Assemble2DSlice(const ezTexConvSliceChannelMapping& mapping, ezUInt32 uiResolutionX, ezUInt32 uiResolutionY, ezColor* pPixelOut) const;
-
-  enum class MipmapChannelMode
```

Comment:
here as well, this change was introduced on the ez side

Reply:
fixed